packages/liveblocks-devtools/src/devtools/tabs/presence.tsx

```diff
@@ -13,7 +13,11 @@ export function Presence({ className, ...props }: ComponentProps<"div">) {
   const others = useOthers();
   const presence = useMemo(() => (me ? [me, ...others] : others), [me, others]);
 
-  if (currentStatus === "open") {
+  if (
+    currentStatus === "connected" ||
+    currentStatus === "open" || // Same as "connected", but only sent by old clients (prior to 1.1)
+    currentStatus === "reconnecting"
```

Comment:
Now that we can differentiate between "connecting" and "reconnecting", it's perfectly fine to render the Storage/Presence views while still reconnecting! It's a much nicer experience than to see the views jump back to "loading" again while reconnecting, like it was doing before.